src/celeritas/ext/detail/GeantGeometryImporter.hh

```diff
@@ -0,0 +1,145 @@
+//----------------------------------*-C++-*----------------------------------//
+// SPDX-FileCopyrightText: 2020 CERN
+// SPDX-License-Identifier: Apache-2.0
+//---------------------------------------------------------------------------//
+/*!
+ * \file GeantGeometryImporter.hh
+ * \brief Class to create a VecGeom model from a pre-existing Geant4 geometry
+ *
+ * Original code from G4VecGeomNav package by John Apostolakis et.al.
+ *
+ * Original source:
+ * https://gitlab.cern.ch/VecGeom/g4vecgeomnav/-/raw/fdd310842fa71c58b3d99646159ef1993a0366b0/include/G4VecGeomConverter.h
+ */
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <utility>
+#include <vector>
+#include <G4PVReplica.hh>
+#include <G4RotationMatrix.hh>
+#include <G4ThreeVector.hh>
+#include <VecGeom/base/TypeMap.h>
+#include <VecGeom/management/GeoManager.h>
+
+class G4LogicalVolume;
+class G4AffineTransformation;
+class G4VPhysicalVolume;
+class G4VSolid;
+
+namespace celeritas
+{
+
+// class converting G4 to VecGeom (only geometry; no materials)
+using vecgeom::BidirectionalTypeMap;
+using vecgeom::LogicalVolume;
+using vecgeom::Transformation3D;
+using vecgeom::VPlacedVolume;
+using vecgeom::VUnplacedVolume;
```

Comment:
These shouldn't be in the celeritas namespace

Reply:
Moved them inside the class definition.